_includes/animated-illustration-footer.html

```diff
@@ -1,3 +1,20 @@
-<script type="text/javascript" src="https://himaxwell.net/wp-content/themes/Maxwell/resources/js/parts/cali2.js?ver=5.8.1" id="maxwell-lottie2-js"></script>
-<script type="text/javascript" src="https://himaxwell.net/wp-content/themes/Maxwell/resources/js/parts/sw2.js?ver=5.8.1" id="maxwell-lottie4-js"></script>
-<script type="text/javascript" src="https://himaxwell.net/wp-content/themes/Maxwell/resources/js/parts/ne2.js?ver=5.8.1" id="maxwell-lottie6-js"></script>
\ No newline at end of file
+<script  type="text/javascript">
```

Comment:
👀  Interesting. Why this instead of using the `async` attribute on the `script` tag, or loading at the end of the `body` tag?